app/models/level.rb

```diff
@@ -0,0 +1,17 @@
+# Level
+class Level < ApplicationRecord
+  # has_many :users | need when connecting with the User model
+  # Explain (  level.id is a user_level  )
+
+  validates_presence_of :level_experience, :level_health_point
```

Comment:
please use more straightforward name, for example not `level_experience, level_health_point` but `experience_level, health_point_level`